### PR TITLE
Added Italian README

### PR DESCRIPTION
**Pull request by micheleberardi**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Please delete this note before submitting the pull request.
>
> For 1.15 Features: set Milestone to 1.15 and Base Branch to dev-1.15
> 
> For Chinese localization, base branch to release-1.14
>
> For Korean Localization: set Base Branch to dev-1.14-ko.\<latest team milestone>
>
> Help editing and submitting pull requests:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> Help choosing which branch to use:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

File: README-it.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+* [Inizia a contribuire](https://kubernetes.io/docs/contribute/start/)
+* [Staging Your Documentation Changes](http://kubernetes.io/docs/contribute/intermediate#view-your-changes-locally)
+* [Uso dei modelli di pagina](http://kubernetes.io/docs/contribute/style/page-templates/)
+* [Guida allo stile della documentazione](http://kubernetes.io/docs/contribute/style/style-guide/)
+* [Localizzazione della documentazione di Kubernetes](https://kubernetes.io/docs/contribute/localization/)
+
+## La documentazione di localizzazione di Kubernetes di `README.md`
+
+### Coreano
```
Comment (rlenferink) [inline README-it.md]: Can you rewrite this section from Korean to Italian so it corresponds with the correct maintainers and slack channels. This is what we do for the German docs as well: #14489

File: README-it.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+Il modo consigliato per eseguire localmente il sito Web di Kubernetes è eseguire un'immagine specializzata [Docker](https://docker.com) che include il generatore di siti statici [Hugo](https://gohugo.io).
+
+> Se si utilizza Windows, sono necessari alcuni altri strumenti che è possibile installare con [Chocolatey](https://chocolatey.org). `choco install make`
+
+> Se preferisci eseguire il sito Web localmente senza Docker, vedi [Esegui localmente il sito usando Hugo](# running-the-site-local-using-hugo) di seguito.
+
+Se hai Docker [attivo e funzionante](https://www.docker.com/get-started), crea l'immagine del Docker `kubernetes-hugo` localmente:
+
+```Bash
```
Comment (rlenferink) [inline README-it.md]: Lowercase
```suggestion
```bash
```

File: README-it.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+make docker-image
+```
+
+Una volta che l'immagine è stata creata, puoi eseguire il sito localmente:
+
+```Bash
```
Comment (rlenferink) [inline README-it.md]: Lowercase
```suggestion
```bash
```

File: README-it.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+Aprire il browser a http://localhost:1313 per visualizzare il sito. Man mano che si apportano modifiche ai file sorgente, Hugo aggiorna il sito e forza l'aggiornamento del browser.
+
+## Eseguire il sito localmente usando Hugo
+
+Vedere la [documentazione ufficiale di Hugo](https://gohugo.io/getting-started/installing/) per le istruzioni di installazione di Hugo. Assicurati di installare la versione di Hugo specificata dalla variabile di ambiente `HUGO_VERSION` nel file [`netlify.toml`](netlify.toml#L9). 
+
+Per eseguire il sito localmente quando hai installato Hugo:
+
+```Bash
```
Comment (rlenferink) [inline README-it.md]: Lowercase
```suggestion
```bash
```

File: README-it.md (hunk 2 above)
Comment (micheleberardi) [reply README-it.md]: done

File: README-it.md (hunk 3 above)
Comment (micheleberardi) [reply README-it.md]: done

File: README-it.md (hunk 1 above)
Comment (micheleberardi) [reply README-it.md]: sorry, it's not clear this part

Comment (rlenferink) [reply README-it.md]: This section is written in Italian, but is describing the Korean process.
I was aiming at rewriting this to something as:

```text
### Italian

......

Maintainers:

* Michele Berardi ([@micheleberardi](https://github.com/micheleberardi))

Slack channel: .....
```

I think you can keep the translated part but then update the data (names, usernames, link to kubernetes-docs-ko) to explain the Italian structure.

File: README-it.md (hunk 2 above)
Comment (rlenferink) [reply README-it.md]: I couldn't find any new commit and the first commit is still containing the uppercase variant, therefore I'm 'unresolving' these ones.

File: README-it.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
+# La documentazione di Kubernetes
+
+[![Build Status](https://api.travis-ci.org/kubernetes/website.svg?branch=master)](https://travis-ci.org/kubernetes/website)
```
Comment (sftim) [inline README-it.md]: “Build Status” is in English still; should that be localized?

File: README-it.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+Questo avvierà il server Hugo locale sulla porta 1313. Apri il browser a http://localhost:1313 per visualizzare il sito. Man mano che si apportano modifiche ai file sorgente, Hugo aggiorna il sito e forza l'aggiornamento del browser.
+
+## Comunità, discussioni, contributi e supporto
+
+Scopri come interagire con la community di Kubernetes sulla [pagina della community](http://kubernetes.io/community/).
+
+Puoi raggiungere i manutentori di questo progetto su:
+
+- [Slack](https://kubernetes.slack.com/messages/sig-docs)
+- [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-docs)
```
Comment (sftim) [inline README-it.md]: > Mailing List
Does this term localize?

**Comment by rlenferink** [comment]
ping @micheleberardi

**Pull request closed without merging**